### ISSUE TITLE
defaultRegistry option is confusing

### ISSUE THREAD

**Issue by fiveisprime** [Microsoft/vscode-docker#210]
What is the intent of `defaultRegistry`? I was expecting it to prepend the registry when building and it only seems to work with tagging. Is this by design?

**Comment by chrisdias** [comment]
yes it is meant for tagging, not building.  my thinking was that you build often and don't want to deal with a long image name.  

when you deploy, you tag the image with the registry name plus a version and then deploy.  this seems like a much less frequent task (like in github, i only tag on a release) so it only applies there.

if this is an incorrect assumption i'm happy to revisit.

**Comment by fiveisprime** [comment]
Ahhh, I understand your thought process here, makes sense.

I think the solution is docs and/or a better tooltip

![image](https://user-images.githubusercontent.com/1186948/35832462-424fdb3a-0a82-11e8-9ec7-673b200e28a6.png)

I'll send you a PR.

**Comment by chrisdias** [comment]
https://github.com/Microsoft/vscode-docker/blob/4f0d9c9fa437a377b829ce5eaf72fb5367b6681b/package.json#L305
